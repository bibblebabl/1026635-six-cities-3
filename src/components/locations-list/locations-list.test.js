import React from 'react';
import renderer from 'react-test-renderer';

import LocationsList from './locations-list';

const cities = [`Amsterdam`, `Hamburg`, `Brussels`, `Cologne`];

const props = {
  cities,
  selectedCity: {
    "name": `Amsterdam`,
    "location": {
      "x": 52.37454,
      "y": 4.897976,
      "zoom": 13
    }
  },
  onCityNameClick: jest.fn()
};

it(`<LocationsList /> renders correctly`, () => {
  const component = renderer.create(
      <LocationsList {...props} />,
      {
        createNodeMock: () => document.createElement(`div`)
      })
  .toJSON();

  expect(component).toMatchSnapshot();
});
