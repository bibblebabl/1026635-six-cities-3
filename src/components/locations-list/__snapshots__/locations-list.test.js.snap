// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LocationsList /> renders correctly 1`] = `
<ul
  className="locations__list tabs__list"
>
  <a
    className="locations__item-link tabs__item "
    href="#"
    onClick={[Function]}
  >
    <span>
      Cologne
    </span>
  </a>
  <a
    className="locations__item-link tabs__item "
    href="#"
    onClick={[Function]}
  >
    <span>
      Amsterdam
    </span>
  </a>
  <a
    className="locations__item-link tabs__item "
    href="#"
    onClick={[Function]}
  >
    <span>
      Paris
    </span>
  </a>
  <a
    className="locations__item-link tabs__item "
    href="#"
    onClick={[Function]}
  >
    <span>
      Hamburg
    </span>
  </a>
</ul>
`;
